modules/API/pages/gsql-endpoints.adoc

```diff
@@ -0,0 +1,4415 @@
+----
+--
+====
+==== Parameters:
+
+//[width="100%",cols="25%,25%,50%",options="header",]
+[%autowidth]
+|===
+| Name | Required | Description
+
+| `gsql`
+| yes
+| this should be true, means gsql command foramt
+|===
+
+
+'''     
+
+=== create vertex (add global vertex to local graph)
+
+`POST /gsql/v1/schema/vertices`
+
+This endpoint is used to add global vertices to local graph
+
+==== Example
+
+[,tabs]
+====
+Request::
++
+--
+[source,bash]
+----
+curl -X POST -H "content-type: application/json" "http://localhost:14240/gsql/v1/schema/vertices?gsql=false&graph=empty_graph" -d '{"addVertices":["user","website"]}'
+----
+--
+Response::
++
+--
+[source.wrap,console]
+----
+{"error":false,"message":"Successfully add vertices: [user, website] on graph empty_graph"}
+----
+--
+====
+
+==== Parameters:
+
+//[width="100%",cols="25%,25%,50%",options="header",]
+[%autowidth]
+|===
+| Name | Required | Description
+
+| `gsql`
+| yes
+| this should be false, means a json format
+
+| `graph`
+| yes
+| which graph the global vertex to be added
+|===
+
+'''   
+=== create vertex (create vertex in global level)
+
+`POST /gsql/v1/schema/vertices`
+
+This api is used to create global vertex using json 
+
+==== Example
+
+[,tabs]
+====
+Request::
++
+--
+[source,bash]
+----
+curl -X POST -H "content-type: application/json" "http://localhost:14240/gsql/v1/schema/vertices?gsql=false" -d '{"createVertices":[{"Config":{"STATS":"OUTDEGREE_BY_EDGETYPE"},"Attributes":[{"AttributeType":{"Name":"STRING"},"AttributeName":"name"}],"PrimaryId":{"AttributeType":{"Name":"UINT"},"AttributeName":"user_id"},"Name":"User5"},
+{"Config":{"STATS":"OUTDEGREE_BY_EDGETYPE"},"Attributes":[{"AttributeType":{"Name":"STRING"},"AttributeName":"name"}],"PrimaryId":{"AttributeType":{"Name":"UINT"},"AttributeName":"user_id"},"Name":"User4"}
+]}'
+----
+--
+Response::
++
+--
+[source.wrap,console]
+----
+{"error":false,"message":"Successfully create vertices: [User5, User4]"}
+----
+--
+====
+
+==== Parameters:
+
+//[width="100%",cols="25%,25%,50%",options="header",]
+[%autowidth]
+|===
+| Name | Required | Description
+
+| `gsql`
+| yes
+| this should be false, means a json format
+|===
+
+'''  
+
+=== drop global vertices
+
+`DELETE /gsql/v1/schema/vertices`
+
+This endpoint is used to drop global vertices.
+
+==== Example
+
+[,tabs]
+====
+Request::
++
+--
+[source,bash]
+----
+curl -X DELETE -H "content-type: text/plain" "http://localhost:14240/gsql/v1/schema/vertices?vertexName=user2,user3"
+----
+--
+Response::
++
+--
+[source.wrap,console]
+----
+{"error":false,"message":"Vertices [user2, user3] deleted successfully."}
+----
+--
+====
+
+==== Parameters:
+//[width="100%",cols="25%,25%,50%",options="header",]
+[%autowidth]
+|===
+| Name | Required | Description
+
+| `vertexName`
+| yes
+| the vertex types to be delete, if there exist more than one, use , separate it. value all means dorp all the vertices
+|===
+
+'''  
+
+=== drop local vertices
+
+`DELETE /gsql/v1/schema/vertices`

```

Comment:
What's the difference between this one and `drop global vertices`? If the API are the same, we'd better combine them into one.